src/backends/webgl.ts

```diff
@@ -301,25 +286,53 @@ let compile_rules: ASTVisit<Emitter, string> =
           throw "dynamic `vtx` calls unimplemented";
         }
 
-      // And our intrinsic for indicating the rendering stage.
+        // And our intrinsic for indicating the rendering stage.
       } else if (render_expr(tree)) {
         // Pass through the code argument.
         return emit(emitter, tree.args[0]);
+      } else if (tree.fun.tag === "lookup") {
+        // Emit arguments
+        let args: string[] = [];
+        let argsType: Type[] = [];
+        for (let arg of tree.args) {
+          args.push(paren(emit(emitter, arg)));
+          let [typ,] = emitter.ir.type_table[arg.id!];
+          argsType.push(typ);
+        }
+        let func = (tree.fun as ast.LookupNode).ident;
+
+        let res = getFunc(func, argsType, args);
```

Comment:
A comment here to explain that we're looking up an intrinsic WebGL function might be helpful here. (The same is true for the calls to `getFunc` in the binary and unary cases.)